### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by doc-E-brown**
## Related Tasks

N/A

## Depends on

N/A

## What

Update the README.md documentation

## Why

Explanation of the repo and documentation is required for this to be useful.

## Concerns

N/A

## Notes
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,155 @@
 # Data Engineering Handbook
 
-This repo contains the contents of the [harrison.ai](https://harrison.ai) Data Engineering Handbook which is published via GitHub Pages at [https://harrison-ai.github.io/dataeng-handbook/](https://harrison-ai.github.io/dataeng-handbook/). This handbook describes the standard processes being used by the harrison.ai Data Engineering team.
+This repo contains the contents of the [harrison.ai](https://harrison.ai) Data Engineering Handbook which is published via GitHub Pages at [https://dataengineering.harrisonai.io](https://dataengineering.harrisonai.io). This handbook describes the standard processes being used by the harrison.ai Data Engineering team and is intended to be a collaborative dynamic document that is contributed to by all members of the data engineering team.  This handbook and the corresponding repo has intentionally been made public for a number of different reasons:
 
-## Development
+* our hope is that this document may be used by others in the data engineering space as a reference or inspiration for their own methods and processes;
```
Comment (oliverdaff) [inline README.md:5]: Should this line end with a `.` rather than a `;`?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,155 @@
...
+* it can be referred to and used by engineers looking to join the harrison.ai data engineering team, particularly during the interview / recruitment process.
+* it is simply easier to access documentation that does not need to be protected via a standard URL.
 
-With docker and docker-compose installed you can run a development instance of the Github Pages site using:
+This handbook is licensed under the [Mozilla Public License](LICENSE) so it can be easily forked and modified without the requirement to contribute back to this project.
 
+# Documentation Structure
+
+This repo has been structured to allow for this document to be easily forked and customised for a number of different use cases.  As such there are a few key components and design choices that
```
Comment (oliverdaff) [inline README.md:13]: Could this be changed to `This repo has been structured to allow this documentation ... `?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,155 @@
...
+├── publishconf.py
+├── README.md
+├── requirements.txt
+└── tasks.py
+```
+
+As the handbook is built using the [Pelican](https://docs.getpelican.com/en/latest/) static site generator, the content of the handbook is to be written using [markdown](https://www.markdownguide.org/) and placed in the **content** directory.
+If you wish to use a sub-directory to logically group content you can do that too, just ensure you have an **index.md** file in the sub-directory as the main page or index page for the group.
+
+The other key component of this structure is the **pelican-theme** directory, or in this case git submodule.  The theme is installed as a git submodule as it allows for the theming and styling decisions within the handbook to be changed easily if one wishes to create a fork of the handbook and apply their own custom theming.
```
Comment (oliverdaff) [inline README.md:59]: Can we add a link to git submodules incase readers are not familar with them?

File: README.md [hunk 4]
```diff
@@ -1,11 +1,155 @@
...
+In some instances there may be a need to keep some content private, but not maintain two distinct copies.  In this case it is recommended that a fork of the public handbook be made and be used as the upstream source.  In that way the private content can be added to the **content** folder of the fork using either different filenames or sub-directories.
+If an update is made to the public repository then the changes can equally be applied to the downstream fork simply by rebasing or pulling from upstream.
+
+![structure diagram](handbookstructure.png)
+
+
+# Development
+
+If you do not intend to serve or view the content of this handbook locally, but rather write and submit content then all you need is git and a text editor to draft the markdown documentation.
+To make a new page copy and rename the file **PAGE_TEMPLATE.md** and paste it into the **content** directory.  When renaming the file it .  Remember if you are using a sub-directory you will need at least one copy of the **PAGE_TEMPLATE.md** to be named **index.md**.
```
Comment (oliverdaff) [inline README.md:73]: I think `When renaming the file . Remember` should be one sentence.

**Review by oliverdaff — CHANGES_REQUESTED** [review]
This a clear explanation of how this repo is intended to be used :clap: .
There are a couple of changes I think are needed but just proof reading rather than structure or content.

File: README.md (hunk 1 above)
Comment (doc-E-brown) [reply README.md:5]: yes

**Review by oliverdaff — APPROVED** [review]
:+1:

**Pull request merged**